x/pool-incentives/keeper/genesis.go

```diff
@@ -9,7 +9,12 @@ import (
 func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
 	k.SetLockableDurations(ctx, genState.LockableDurations)
-	if genState.DistrInfo != nil {
+	if genState.DistrInfo == nil {
+		k.SetDistrInfo(ctx, types.DistrInfo{
+			TotalWeight: sdk.NewInt(0),
+			Records:     nil,
+		})
+	} else {
```

Comment:
Doing this to remove initializing distrInfoKey logic that was in GetDistrInfo

Reply:
LGTM!